src/api/getBridgedBlocks.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+import { ApiPromise } from '@polkadot/api';
+import { VoidFn } from '@polkadot/api/types';
+import { useGetApi, ReturnApi } from './useGetApi';
+
+interface DataType {
+  api: ApiPromise;
+  apiMethod: string;
+  separator: string;
```

Comment:
If this is expected to be only two values maybe better do:
```suggestion
  separator: 'bestFinalized' | 'importedHeaders';
```
?

Reply:
Or even better let's just have two separate functions?

Reply:
Im trying to figure out if there is a way of a reusable hook/function that will bind Api usage with ApiSubscriptions together. Noted your comment though